src/config/index.js

```diff
@@ -0,0 +1,4 @@
+export const BASE_URL = 'https://team-odd-bn-backend-staging.herokuapp.com/api/v1';
+export const ICON = 'https://res.cloudinary.com/hezzie/image/upload/v1577645239/Screen_Shot_2019-12-29_at_20.46.06_vklvux.png';
+export const FACEBOOK_CALLBACK_URL = 'https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/facebook';
+export const GOOGLE_CALLBACK_URL = 'https://team-odd-bn-backend-staging.herokuapp.com/api/v1/auth/google';
```

Comment:
This `BASE_UR, ICON,  FACEBOOK_CALLBACK_URL and GOOGLE_CALLBACK_URL` could be better when it's placed in `.env` file

Reply:
I have tried to save all credentials in `.env` but I could not be able to access them on Heroku.

I started a thread [Thread](https://andela.slack.com/archives/GQ058HTME/p1577718776021600?thread_ts=1577718776.021600) but no answer yet.
Do you have another approach to help?